src/K1DModel/parameters.jl

```diff
@@ -1,25 +1,62 @@
-"""
-    Create Kinematic1D model parameters.
-
-    (Overwriting the default values happens in the driver file.
-    Here we are just creating the Kinematic1DParameters struct)
-"""
 module Parameters
 
+using DocStringExtensions
+import ClimaParams as CP
+
 abstract type AbstractKinematicParameters end
 const AKP = AbstractKinematicParameters
 
-# Define KiD parameters
+"""
+    Kinematic1DParameters{FT}
+
+    Free parameters for the kinematic 1-dimensional simulation
+
+    #Fields
+    $(DocStringExtensions.FIELDS)
+"""
 Base.@kwdef struct Kinematic1DParameters{FT} <: AKP
+    "Maximum updraft momentum flux m/s kg/m3"
     w1::FT
+    "Time when the updraft is switched off [s]"
     t1::FT
+    "Surface pressure [Pa]"
     p0::FT
+    "Switch to include flux correction for moisture transport"
     qtot_flux_correction::Int
+    "Assumed aerosol dry radius [m]"
     r_dry::FT
+    "Assumed aerosol standard deviation [-]"
     std_dry::FT
+    "Assumed aerosol hygroscopicity"
     κ::FT
 end
 
+function Kinematic1DParameters(td::CP.AbstractTOMLDict)
+    #name_map = (;
```

Comment:
@nefrathenrici - could you take a look at why the map didn't work here?

Reply:
I think on line 46 you need to splat the parameters: `return Kinematic1DParameters{FT}(; parameters...)`